### PR TITLE
Add Phase-2 Pixel LA payload creation code, config and unit test

### PR DESCRIPTION
**Pull request by tvami**
#### PR description:

Add Phase-2 Pixel LA payload creation code, config and unit test. The config is an example for 3D sensor containing case, which is the T25 in CMSSW (T33 is the latest but that's not in CMSSW yet, and anyway it's not as important for the unit test).

#### PR validation:

Added a dedicated unit test and ran `scramv1 b runtests`

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Not a backport, and no backport is needed.

### CODE REVIEW

**Comment by tvami** [comment]
@cmsbuild , please test

File: CondTools/SiPixel/plugins/SiPixelLorentzAngleDBLoader.cc [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+      int side = tTopo->side(detid);  // 1:-z 2:+z for fpix, for bpix gives 0
+
+      // fill bpix values for LA
+      if (detid.subdetId() == static_cast<int>(PixelSubdetector::PixelBarrel)) {
+        int layer = tTopo->pxbLayer(detid);
+        // Barrel ladder id 1-20,32,44.
+        int ladder = tTopo->pxbLadder(detid);
+        // Barrel Z-index=1,8
+        int module = tTopo->pxbModule(detid);
+        if (module < 5) {
```
Comment (mmusich) [inline CondTools/SiPixel/plugins/SiPixelLorentzAngleDBLoader.cc:94]: magic number?

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by saumyaphor4252** [comment]
+1
- unite tests pass
- diffs only in MsgLogger

**Comment by perrotta** [comment]
+1

**Pull request merged**